docs/Configuration.md

```diff
@@ -24,6 +24,14 @@ token = "secrettoken"
 dir = "/tmp/.cache/sccache"
 size = 7516192768 # 7 GiBytes
 
+[cache.disk.direct_mode]
+use_direct_mode = true
+file_stat_matches = true
+use_ctime_for_stat = true
+ignore_time_macros = false
+skip_system_headers = false
+hash_working_directory = true
```

Comment:
Could you a description what impact each of these has above eache entry, and a reason when to / not to activate / deactivate?

Reply:
I was worried of the duplication of this information. I have it in the struct docstring, then in `Local.md`, and now here. Maybe I could move it from `Local.md` to `Configuration.md`, or just add a comment pointing from `Configuration.md` to `Local.md`? What do you think?

Reply:
I agree, but we should have it preferably in all cases, there might be a bit shorter in one or the other documentation scope.